src/memphis.ts

```diff
@@ -247,14 +253,14 @@ class Memphis {
           for await (const s of this.brokerManager.status()) {
             switch (s.type) {
               case 'update':
-                console.log(`reconnected to memphis successfully`);
+                this.logger.info(`reconnected to memphis successfully`);
```

Comment:
Instead using the new package, please use if statements based on the new flag you have added